docs/src/content/docs/es/showcase.mdx

```diff
@@ -18,11 +18,36 @@ Starlight ya está siendo utilizado en producción. Estos son algunos de los sit
 
 Consulta todos los [repositorios de proyectos públicos que utilizan Starlight en GitHub](https://github.com/withastro/starlight/network/dependents).
 
-## Plugins de la comunidad
+## Plugins
+
+Los [Plugins](/es/reference/plugins) pueden personalizar la configuración, UI y el comportamiento de Starlight, al mismo tiempo que son fáciles de compartir y reutilizar.
+Extiende tu sitio con plugins oficiales respaldados por el equipo de Starlight y plugins de la comunidad mantenidos por usuarios de Starlight.
+
+### Plugins oficiales
+
+<CardGrid>
+	<LinkCard
+		href="/guides/site-search/#algolia-docsearch"
```

Comment:
One detail here, this link should be to the Spanish page:

```suggestion
		href="/es/guides/site-search/#algolia-docsearch"
```

Easy to miss with the format so different from links.